test-requirements.txt

```diff
@@ -1,6 +1,6 @@
 pip>=21.3.1
 wheel>=0.33.1
-pytest>=5.2.0
+pytest==7.0.1
```

Comment:
Just curious - are we doing something that requires us to use this specific version of pytest? If we need to upgrade to version 7, can we make this `pytest>=7.0.1` instead?